app/Http/Controllers/Api/v1/StatementMultipleAPIController.php

```diff
@@ -41,10 +41,6 @@ public function __construct(
      */
     public function store(Request $request): JsonResponse
     {
-        if (!$request->user()->platform || !$request->user()->can('create statements')) {
```

Comment:
@alainvd this one is not so straightforward. I took it out because ideally, it's a redundant check, but we should have something in user create and edit that if they have the contributor role or admin role, then they need to have a platform. The only way I could test this method was to force the scenario unrealistically in code.